app/scripts/seller-transform.js

```diff
@@ -15,13 +15,56 @@ var sellerTransform = (function(_, relatedEntityTransform, commonTransforms) {
 
             if(data.hits.hits.length > 0) {
                 newData._id = _.get(data.hits.hits[0], '_id');
-                newData.telephone = _.get(data.hits.hits[0]._source, 'telephone.name');
-                newData.emailAddress = _.get(data.hits.hits[0]._source, 'email.name');
-                newData.title = 'Seller (' + (newData.telephone || newData.emailAddress || 'Info N/A') + ')';
+                newData._type = _.get(data.hits.hits[0], '_type');
+                newData.telephone = commonTransforms.getClickableObjectArr(_.get(data.hits.hits[0]._source, 'telephone'), 'phone');
+                newData.emailAddress = commonTransforms.getClickableObjectArr(_.get(data.hits.hits[0]._source, 'email'), 'email');
+                var title = undefined;
+                var numPhoneEmails = 0;
+                if(newData.telephone.length > 0) {
+                    title = newData.telephone[0].title;
+                    numPhoneEmails += newData.telephone.length;
+                }
+                if(newData.emailAddress.length > 0) {
+                    title += ", " + newData.emailAddress[0].title;
```

Comment:
The "title += ..." would result in undesirable behavior if newData.telephone was empty (and title was still undefined).